### PR TITLE
Added confirm message if learner data is present [#95537580]

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

File: spec/javascripts/itsi_editor_spec.coffee [hunk 1]
```diff
@@ -144,6 +145,27 @@ describe "ITSI Editor", () ->
 
       expect(children.style.display).toBe("block")
 
+    it "toggles does not children visibility when not confirmed", ->
```
Comment (knowuh) [inline spec/javascripts/itsi_editor_spec.coffee]: Coder Yoda is.

File: spec/javascripts/itsi_editor_spec.coffee [hunk 2]
```diff
@@ -144,6 +145,27 @@ describe "ITSI Editor", () ->
...
+      checkbox = jasmine.react.findTag sectionEditorElement, "input"
+      children = (jasmine.react.findClass sectionEditorElement, "ia-section-editor-elements").getDOMNode()
+
+      expect(children.style.display).toBe("block")
+
+      request = jasmine.react.captureRequest ->
+        jasmine.react.change checkbox, {"target": {"checked": false}}
+      expect(request).toBe(undefined)
+
+      expect(children.style.display).toBe("block")
```
Comment (knowuh) [inline spec/javascripts/itsi_editor_spec.coffee]: We might also assert that when alert is confirmed/dismissed, that the block is hidden right?  I am not sure if there are good examples for how to do this.  I found this SO discussion http://stackoverflow.com/questions/19519328/testing-javascript-alerts-with-jasmine  Its probably fine to punt on this though.

**Comment by knowuh** [comment]
Looks good.  :+1:

File: spec/javascripts/itsi_editor_spec.coffee (hunk 2 above)
Comment (dougmartin) [inline spec/javascripts/itsi_editor_spec.coffee]: The ```confirmHide: -> false``` in the props simulates the alert being dismissed.

File: spec/javascripts/itsi_editor_spec.coffee [hunk 3]
```diff
@@ -121,6 +121,7 @@ describe "ITSI Editor", () ->
       props =
         title: "test title"
         toHide: "hidden"
+        confirmHide: -> true
```
Comment (dougmartin) [inline spec/javascripts/itsi_editor_spec.coffee]: This simulates confirming the hide.

**Pull request merged**